docs-kits/kits/behaviour-twin-kit/overview.md

```diff
@@ -0,0 +1,37 @@
+﻿---
```

Comment:
@AngelikaWittek: Can you please help if this PR needs an Eclipse IP check (e.g. over 1000 loc)?

Background: Two KITs were merged into one KIT. Therefore, nothing has changed in terms of content - only the structure has been restructured

Reply:
There are some additional/revised code blocks (e.g. skills). These are provided by the Agents KIT (the new/next Agents KIT version 24-05 will contain all the examples). We added the contributors at the legal notice(s) and they are mentioned in the headers of the code blocks.